src/common/profile-validator/profile-validator.service.ts

```
@@ -0,0 +1,232 @@
+    }
+  }
+
+  private getNewAttributeValue(
+    vc: any,
+    attributeValue: any,
+    nameFieldsPosition: any,
+    profileAttribute: any,
+  ) {
+    switch (vc.vcType) {
+      case 'digilocker':
+        const nameValues: string[] = attributeValue.split(' ');
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)